src/app/main/time-groups/edit-time-group/edit-time-group.component.ts

```diff
@@ -119,16 +119,6 @@ export class EditTimeGroupComponent implements OnInit {
     this.timeGroup.time_group_schedule_attributes!.schedule_days_attributes != scheduleDays;
   }
 
-  public updateRule(){
```

Comment:
Please remove it also form the tine group service

Reply:
done